File: trail1/남녀노소 구분짓기/sex-and-age.py
S = int(input())
Y = int(input())

if S == 0 and Y >= 19:
    print("MAN")
elif S == 1 and Y >= 19:
    print("WOMAN")
elif S == 1 and Y < 19:
    print("GIRL")
else:
    print("BOY")

File: trail1/남녀노소 구분짓기/sex-and-age.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int S = sc.nextInt();
        int Y = sc.nextInt();
        if (S == 0 && Y >= 19) {System.out.println("MAN");}
        else if (S == 1 && Y >= 19) {System.out.println("WOMAN");}
        else if (S == 1 && Y < 19) {System.out.println("GIRL");}
        else {System.out.println("BOY");}
    }
}